pkg/vpcmodel/grouping.go

```diff
@@ -288,6 +319,20 @@ func (g *GroupConnLines) String() string {
 	return strings.Join(linesStr, "\n") + asteriskDetails
 }
 
+// get the grouped connectivity output
+func (g *GroupConnLines) StringForDrawio() string {
+	linesStr := make([]string, len(g.GroupedLines))
+	for i, line := range g.GroupedLines {
+		if line.AllowAllConn {
+			linesStr[i] = line.Src.Name() + implies + line.Dst.Name() // do not print connection string
+		} else {
+			linesStr[i] = line.String()
+		}
+	}
+	sort.Strings(linesStr)
+	return strings.Join(linesStr, "\n")
+}
+
 // StringTmpWA ToDo: tmp WA until https://github.com/np-guard/vpc-network-config-analyzer/issues/138.
```

Comment:
not shore what StringForDrawio() is for

Reply:
`StringForDrawio()` is supposed to be the `nice connection name, without "protocol:"` you asked for, but now I see it does something else; will fix.

Reply:
Looking at the code I think more details are needed @haim-kermany.
1. Printing the "connection":  is done by the following function, and is the same with and without `grouping`. Perhaps what is required is a dedicated printing function for `drawio` which I can use also for `grouping`; how do you handle the same issue for the non grouping case?
`func (conn *ConnectionSet) String() string {`

2. What exactly do you mean by `nice connection name, without "protocol:`?
 This is where "protocol:" comes from. Note that there are other stuff there other than "protocol"
 Please define accurately what is it you want to print in connections. 
```
 func getDimensionStr(dimValue *CanonicalIntervalSet, dim Dimension) string {
	domainValues := getDimensionDomain(dim)
	if dimValue.Equal(*domainValues) {
		// avoid adding dimension str on full dimension values
		return ""
	}
	switch dim {
	case protocol:
		pList := []string{}
		for p := minProtocol; p <= maxProtocol; p++ {
			if dimValue.Contains(p) {
				pList = append(pList, getProtocolStr(p))
			}
		}
		return "protocol: " + strings.Join(pList, ",")
	case srcPort:
		return "src-ports: " + dimValue.String()
	case dstPort:
		return "dst-ports: " + dimValue.String()
	case icmpType:
		return "icmp-type: " + dimValue.String()
	case icmpCode:
		return "icmp-code: " + dimValue.String()
	}
	return ""
}
```